src/code/mempak.c

```diff
@@ -1,3 +1,7 @@
+#include "config.h"
+
+#ifdef ENABLE_CAMERA_DEBUGGER
```

Comment:
Can't the mempak technically be used for other things? Might be better not to remove these entirely just because it's not used by the vanilla game. Also it would be better to ifdef this in the spec instead of in the body of the file, imo.

Reply:
afaik the mempak is only used for the cutscene exporter, when I did this I only thought about removing each debug feature and wasn't thinking about another use case, I'll change that and make its own define for the mempak it seems the best option 👍 

(so yea technically it could be used for something else, though idk how you'd use it, I don't even know how are you supposed to get a mempak's data)